src/Program.spec.ts

```diff
@@ -185,6 +186,46 @@ describe('Program', () => {
     });
 
     describe('validate', () => {
+        it('catches duplicate XML component names', async () => {
+            //add 2 components which both reference the same errored file
+            await program.addOrReplaceFile({ src: `${rootDir}/components/component1.xml`, dest: 'components/component1.xml' }, `
+                <?xml version="1.0" encoding="utf-8" ?>
+                <component name="Component1" extends="Scene">
+                </component>
+            `);
+            await program.addOrReplaceFile({ src: `${rootDir}/components/component2.xml`, dest: 'components/component2.xml' }, `
+                <?xml version="1.0" encoding="utf-8" ?>
+                <component name="Component1" extends="Scene" extends="Scene">
```

Comment:
Not sure if that's even valid xml with the double extends

Reply:
oops. copy-paste error.